### DISCUSSION TITLE
Hints from the glossary based on the key

### DISCUSSION THREAD

**Discussion by zsuatem** [WeblateOrg/weblate#10640] [Q&A]
Is it possible to configure Weblate so that for translated phrases the glossary prompts are suggested based on the phrase key and not just the phrase itself?

I think in some older version something like that worked, but now I can't find it.

E.g..
I have a phrase "to translate" to which corresponds the key "mystring-abc". I would now like to have an entry for "abc" in the glossary appear the suggestion for the phrase "to translate".

**Comment by nijel** [comment]
I don't think glossary ever worked based on the keys. Actually, you typically don't have the keys present in glossaries.
